examples/supermassive-todomvc/src/components/TodoList.tsx

```diff
@@ -3,6 +3,7 @@ import React from "react";
 import { useLazyLoadQuery } from "@graphitation/apollo-react-relay-duct-tape";
 import { graphql } from "@graphitation/graphql-js-tag";
 import Todo, { TodoFragment } from "./Todo";
+import { TodoListQuery as TodoListQueryType } from "./__generated__/TodoListQuery.graphql";
```

Comment:
This renaming is only necessary right now because of storing the GraphQL doc in a module-scope variable. Instead, can you inline the document at the `useLazyLoadQuery` call-site? Which will be the pattern we want people to actually learn. Then you can remove this renaming too.